packages/rolldown/src/options/input-options.ts

```diff
@@ -139,8 +139,27 @@ export interface InputOptions {
     mainFiles?: string[];
     modules?: string[];
     symlinks?: boolean;
+    /**
+     * - Type: `string | object | undefined`
+     * - Default: `undefined`
+     *
```

Comment:
```suggestion
```

The type in the JSDoc seems not in sync with the actual TS definition (and we can skip it + the default as the types should make it clear already.